### PR TITLE
Removal of unnecessary check causing error in sample disposal emails

### PR DESCRIPTION
**Pull request by EGCGDanForth**
-check preventing multiple projects in step has been removed

### CODE REVIEW

**Comment by EGCGDanForth** [comment]
CR1423

File: scripts/email_sample_disposal_notification.py [hunk 1]
```diff
@@ -7,8 +7,6 @@
 class SampleDisposalNotificationEmail(SendMailEPP):
 
     def _run(self):
-        if len(self.projects) > 1:  # check if more than one project in step, only one permitted
-            raise ValueError('More than one project present in step. Only one project per step permitted')
 
         # Create the message
         msg = 'Hi,\n\nThe samples at the link below have been approved for disposal by the Facility Manager:\n' \
```
Comment (mwhamgenomics) [inline scripts/email_sample_disposal_notification.py]: `sample_count` and `project` below are no longer used in this format string, and can be removed.

File: scripts/email_sample_disposal_review.py [hunk 2]
```diff
@@ -7,8 +7,6 @@
 class SampleDisposalFMEmail(SendMailEPP):
 
     def _run(self):
-        if len(self.projects) > 1:  # check if more than one project in step, only one permitted
-            raise ValueError('More than one project present in step. Only one project per step permitted')
 
         # Create the message
         msg = 'Hi Facility Manager,\n\nSamples are ready for disposal. Please follow the link below and perform the following tasks:\n' \
```
Comment (mwhamgenomics) [inline scripts/email_sample_disposal_review.py]: Same as above

File: tests/test_emails.py [hunk 3]
```diff
@@ -300,6 +300,9 @@ def setUp(self):
         super().setUp()
         self.epp = self.create_epp(SampleDisposalFMEmail)
 
+    def test_only_one_project(self):
+        pass
+
```
Comment (mwhamgenomics) [inline tests/test_emails.py]: ?

File: tests/test_emails.py [hunk 4]
```diff
@@ -334,6 +337,9 @@ def setUp(self):
         super().setUp()
         self.epp = self.create_epp(SampleDisposalNotificationEmail)
 
+    def test_only_one_project(self):
+        pass
+
```
Comment (mwhamgenomics) [inline tests/test_emails.py]: ?

File: tests/test_emails.py (hunk 3 above)
Comment (EGCGDanForth) [reply tests/test_emails.py]: See line 39 of test_emails.py

File: tests/test_emails.py (hunk 4 above)
Comment (EGCGDanForth) [reply tests/test_emails.py]: Default test in test_email needs to be re-defined in child as not required for this script

File: scripts/email_sample_disposal_review.py (hunk 2 above)
Comment (EGCGDanForth) [reply scripts/email_sample_disposal_review.py]: unused variables removed.

**Pull request merged**